input/pagecontent/2-9-GetCurrentContext.md

```diff
@@ -10,34 +10,31 @@ The requester makes an HTTP GET call to the following URL:
 
 GET `base-hub-URL/{topic}`
 
-### Get current context Response
-
-This method returns an object containing the current context of a topic; where the current context is the same event as the most recent open for a given subscriber.  The current context is made up of one or more "top-level" contextual resources and the type of the anchor context in the `context.type` field.  For example, if the current context was established using a [`Patient-open`](3-3-1-patient-open.html) request the returned object will contain `context.type: "Patient"`.  If the current context was created by a [`DiagnosticReport-open`](3-6-1-diagnosticreport-open.html) request the returned object will contain `context.type: "DiagnosticReport"`.
-
+### Get Current Context Response
 
+This method returns an object containing the current context of a topic; where the current context is the same event as the most recent open for a given subscriber.  The current context is made up of one or more "top-level" contextual resources and the type of the anchor context SHALL be in the `context.type` field.  For example, if the current context was established using a [`Patient-open`](3-3-1-patient-open.html) request the returned object will contain `context.type: "Patient"`.  If the current context was created by a [`DiagnosticReport-open`](3-6-1-diagnosticreport-open.html) request the returned object will contain `context.type: "DiagnosticReport"`.  If there is no context currently established the `context.type` SHALL contain an empty string and the `context` SHALL be an empty array.
 
 Field | Optionality | Type | Description
 ---   | --- | --- | ---
-`context.type` | Required | string | ResourceType of the context element.
+`context.type` | Required | string | ResourceType of the context element or an empty string if no context is currently established.
 `context.versionId` | Required | string | The versionId of the current context. Each time the context changes, a different versionId is generated.
-`context`   | Required | array | The context field of the corresponding context element as defined in the `-open` event of the resourceType.
+`context`   | Required | array | The context array of the corresponding context element as was supplied in the most recent `-open` event or an empty array if no context is currently established.

```

Comment:
Discussed during today's FHIRcast call. 

Scenario:
1. imaging-study-open
2. patient-open
3. GetCurretnContext's "most recent" would then return the patient, and nothing about the imaging-study ?

http://build.fhir.org/ig/HL7/fhircast-docs/2-9-GetCurrentContext.html

Solutions:
~~1.  Why are we only theoretically encounting this now? How about we "solve" it by removing these two words introduced in this PR practically unintentionally?~~
~~2. Specify that the "most recent" event is specific to the subscriber.~~
~~3. The hub is now also responsible for "merging" event content somehow?~~
~~4. Subscriber specifies the CurrentContext specific to an anchor resource, that it would like to hear about.~~
5. Replicate language from http://build.fhir.org/ig/HL7/fhircast-docs/2-4-Subscribing.html#current-context-notification-upon-successful-subscription, specifically:
>informing the newly subscribed application of the most recent *-open event for which no *-close event has occurred, according to the app’s subscription

Reply:
Completed